bin/verify-deployment

```diff
@@ -21,8 +21,8 @@ UPDATED_REPLIACS=-1
 REPLICAS=0
 until [ "$UPDATED_REPLIACS" == "$REPLICAS" ]; do
   sleep 1
-  REPLICAS=$(kubectl get deployment ${1} --namespace=${2:=default} -o yaml | grep replicas | sed -n 2p | awk '{print $2}')
-  UPDATED_REPLIACS=$(kubectl get deployment ${1} --namespace=${2:=default} -o yaml | grep updatedReplicas | awk '{print $2}')
+  REPLICAS=$(kubectl get deployment ${1} --namespace=${2:=default} -o yaml | grep replicas: | sed -n 2p | awk '{print $2}')
+  UPDATED_REPLIACS=$(kubectl get deployment ${1} --namespace=${2:=default} -o yaml | grep updatedReplicas: | awk '{print $2}')
```

Comment:
If it works then it looks good. Almost always a fan of more specificity when grepping.